_config.yml

```diff
@@ -12,6 +12,7 @@ description: > # this means to ignore newlines until "baseurl:"
   We do Ruby on Rails and Java applications in Play! Framework or Spring MVC.
 baseurl: "" # the subpath of your site, e.g. /blog
 url: "http://beyondscheme.com" # the base hostname & protocol for your site
+permalink: /:year/:title
```

Comment:
I've changed the permalink pattern to the simpler one. Thanks to that the post url won't have wrong date based on blog post file name where is date 2016-04-26 in _posts/2016-04-26-distributed-game-of-life-in-elixir.md

The old url had pattern /:categories/:year/:month/:day/:title.html
http://localhost:4000/elixir/2016/04/26/distributed-game-of-life-in-elixir.html

The new url:
http://localhost:4000/2016/distributed-game-of-life-in-elixir